testtracer_test.go

```diff
@@ -21,6 +21,7 @@ func nextFakeID() int {
 type testSpanContext struct {
 	HasParent bool
 	FakeID    int
+	Options   *StartSpanOptions
```

Comment:
I'd have a slight preference to just add tags to `testSpan` and test that they get passed through correctly... if you don't feel like implementing this, I can do it :)